docs/source/_static/hackrtd.css

```diff
@@ -110,3 +110,20 @@ a.trio-help-hint, .trio-help-hint a:link, .trio-help-hint a:visited {
     text-decoration: none;
     border-bottom: 1px solid;
 }
+
+/* override table width restrictions */
+@media screen {
+
+   .wy-table-responsive table td {
+      /* !important prevents the common CSS stylesheets from overriding
+         this as on RTD they are loaded after this stylesheet */
```

Comment:
We don't need a comment to explain what !important is here, we're in a CSS file so we assume the readers who don't know what it is can look it up.

Comments are useful when they explain *why* we're doing a change.